beacon-chain/rpc/eth/config/handlers_test.go

```diff
@@ -158,6 +172,9 @@ func TestGetSpec(t *testing.T) {
 	var dam [4]byte
 	copy(dam[:], []byte{'1', '0', '0', '0'})
 	config.DomainApplicationMask = dam
+	var dc [4]byte
+	copy(dc[:], []byte{'1', '1', '0', '0'})
+	config.DomainConsolidation = dc
```

Comment:
I noticed the spec value is `0x0B000000`, so the first 2 bytes being `1` stuck out as wrong. I think this should be `[]byte[11, 0, 0, 0]`: https://go.dev/play/p/fkwg-mzX8Zb

Reply:
These are dummy values. This whole test sucks